app.rb

```diff
@@ -12,26 +12,96 @@ def initialize
     @rentals = []
   end
 
-  def user_input(text)
-    print text
-    gets.chomp
+  class CreateList
+    def user_input(text)
+      print text
+      gets.chomp
+    end
```

Comment:
- [ ] Great job so far but I think first of all after runnimg your app here the app stoped because of user_input is undefined so kindly before using any variable define it before using it 🔥👍🏻✨

### Check the error ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/218726849-0b
6b9680-b99c-4754-a6ea-d117c3239c98.png)